frontend/src/components/Files/RecentActivities.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+interface Activity {
+  user: string;
+  action: "uploaded" | "shared" | "deleted";
```

Comment:
Is the term "deleted" accurate?
I'm assuming we are only deleting the cache version, so maybe something more like "flagged" or "reported"?
It could be misleading users in giving them the impression they can actually delete data...

Reply:
Was more kind of a mock